distributor-api-v1/operations.md

```diff
@@ -1135,8 +1140,81 @@ In case of an error caused by insufficient availability \(which might have decre
+
 
 ### Response <a id="response-7"></a>
 
-Same as in [Create Reservation Group](operations.md#create-reservation-group).
+```json
+{
+    "Id": "f6fa7e62-eb22-4176-bc49-e521d0524dee",
+    "CustomerId": "7ac6ca0d-7c08-4ab1-8da8-9b44979d8855",
+    "Reservations": [],
+    "TotalAmount": { },
+    "PaymentRequestId": "2e3a700a-7b10-4e61-8e9f-acfa00ee00df",
+    "PaymentRequests": [],
+    "Payments": []
+}
+```
+
+This part provide documentation only for the related data in `PaymentRequests` & `Payments` rest of the response is the
+same as in [Create Reservation Group](operations.md#create-reservation-group).
+
+|  | Property | Type | Description |
+| :--- | :--- | :--- | :--- |
+| `PaymentRequests` | Array of [PaymentRequests](operations.md#payment-request) | optional | Payment requests related to the reservation group. |
+| `Payments` | Array of [Payment](operations.md#payments) | optional | Payment attempts related to the Payment requests. |
+
+### PaymentRequest <a id="payment-request"></a>
+
+|  | Property | Type | Description |
+| :--- | :--- | :--- | :--- |
+| `Id` | string | required | Identifier of the payment request. |
+| `ReservationGroupId` | string | required | Id of the related reservation group. |
+| `State` | string [PaymentRequestState](operations.md#payment-request-state) | required | State of the payment request. |
+
+#### PaymentRequestState <a id="payment-request-state"></a>
+
+- `Pending` - Non-finite state. Awaiting a next action.
+- `Completed` - Finite state. Payment request that has been covered by payment.
+- `Canceled` - Finite state. Payment request has been manually canceled by the creator (enterprise).
+- `Expired` - Finite state. Payment request has not been completed withing its expiration time.
+
+### Payment <a id="payment"></a>
+
+|  | Property | Type | Description |
+| :--- | :--- | :--- | :--- |
+| `Id` | string | required | Identifier of the payment. |
+| `Amount` | [SingleCurrencyAmount](operations.md#single-currency-amount) | required | Amount in a currency which was used to create PaymentRequest - usually default currency of the enterprise. |
+| `ChargeAmount` | [SingleCurrencyAmount](operations.md#single-currency-amount) | required | Amount in currency which was used for the payment during the charge. i.e. currency that will be visible on the user bank statement for the payment. |
+| `CreatedUtc` | string | required | Date and time of the payment creation in UTC timezone in ISO 8601 format. |
+| `EnterpriseId` | string | required | Total amount of the reservation. |
+| `PaymentRequestId` | string | required | Identifier of the payment request. |
+| `State` | string [PaymentState](operations.md#payment-state) | required | State of the payment attempt. |
+
+#### SingleCurrencyAmount <a id="single-currency-amount"></a>
+
+An object where name corresponds to ISO code and value represents a structure that holds gross price, net price and tax values.
```

Comment:
We already describe something similar in `Amount` https://mews-systems.gitbook.io/distributor-guide/distributor-api-v1/operations#amount . Maybe that section should be probably rewritten so that it says that `Amount` is an object where keys are currency codes and values are `SingleCurrencyAmounts`. And move the description of individual properties (GrossValue, NetValue, TaxValues) from there here.